coding/point_coding.cpp

```diff
@@ -51,6 +53,15 @@ m2::PointU PointDToPointU(m2::PointD const & pt, uint8_t coordBits)
   return PointDToPointU(pt.x, pt.y, coordBits);
 }
 
+m2::PointU PointDToPointU(m2::PointD const & pt, uint8_t coordBits, m2::RectD const & limitRect)
+{
+  ASSERT_GREATER_OR_EQUAL(coordBits, 1, ());
+  ASSERT_LESS_OR_EQUAL(coordBits, 32, ());
+  auto const x = DoubleToUint32(pt.x, limitRect.minX(), limitRect.maxX(), coordBits);
```

Comment:
assert/check that rect is non-empty
and/or assert/check that min<=max in the called method

Reply:
> check that min<=max in the called function